listen-on-port.js

```diff
@@ -8,8 +8,8 @@ const config = require('./app/config.js')
 const utils = require('./lib/utils.js')
 
 // Set up configuration variables
-const useBrowserSync = config.useBrowserSync.toLowerCase()
-const env = (process.env.NODE_ENV || 'development').toLowerCase()
+var useBrowserSync = config.useBrowserSync.toLowerCase()
+var env = (process.env.NODE_ENV || 'development').toLowerCase()
```

Comment:
This shouldn't change anything, and using `var, when linting, Standard complains.